python/cuspatial/cuspatial/core/trajectory/generic.py

```diff
@@ -1,4 +1,4 @@
-# Copyright (c) 2019, NVIDIA CORPORATION.
+# Copyright (c) 2019-2022, NVIDIA CORPORATION.
```

Comment:
I don't like naming this "generic.py". Why can't you leave it as `trajectory.py`?